types/k6/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for k6 0.31
+// Type definitions for k6 0.32
```

Comment:
Nit: the actual version is `0.32.0`, we shouldn't leave out the patch portion even if it's `0`.